beacon_node/beacon_chain/src/persisted_fork_choice.rs

```diff
@@ -39,4 +42,42 @@ macro_rules! impl_store_item {
 }
 
 impl_store_item!(PersistedForkChoiceV17);
-impl_store_item!(PersistedForkChoiceV28);
+
+impl PersistedForkChoiceV28 {
+    pub fn from_bytes(bytes: &[u8], store_config: &StoreConfig) -> Result<Self, Error> {
+        Self::from_ssz_bytes(&uncompress_bytes(bytes, store_config)?).map_err(Into::into)
+    }
+
+    pub fn as_bytes(&self, store_config: &StoreConfig) -> Result<Vec<u8>, Error> {
+        compress_bytes(&self.as_ssz_bytes(), store_config)
+    }
+
+    pub fn as_kv_store_op(
+        &self,
+        key: Hash256,
+        store_config: &StoreConfig,
+    ) -> Result<KeyValueStoreOp, Error> {
+        Ok(KeyValueStoreOp::PutKeyValue(
+            DBColumn::ForkChoice,
+            key.as_slice().to_vec(),
+            self.as_bytes(store_config)?,
+        ))
+    }
+}
+
+// FIXME(sproul): dedupe?
```

Comment:
Yes it makes sense as this crate already has store as dependency and this function uses StoreConfig

Reply:
fixed